src/ILAMB/Post.py

```diff
@@ -1117,5 +1120,133 @@ def HarvestScalarDatabase(build_dir,filename="scalar_database.csv"):
                             v = grp.variables[vname]
                             V = v[...]
                             s = "nan" if V.mask else "%g" % V
-                            csv += "\n%s,%s,%s,%s,%s,%s,%s,%s,%s,%s" % (category,varname,provider,model,var,g1,region,stype,v.units,s)
+                            csv += "\n" + ",".join(['"%s"' % v for v in (category,varname,provider,model,var,g1,region,stype,v.units,s,weight)])
     with open(os.path.join(build_dir,filename),mode="w") as f: f.write(csv)
+
+def CreateJSON(csv_file,M):
+    """Using the CSV scalar database, create a JSON following the CMEC standard.
+    """
+    def _unCamelCase(s): return re.sub("([a-z])([A-Z])","\g<1> \g<2>",s)
+    def _weightedMean(x): return (x.Data*x.Weight/x.Weight.sum()).sum()
```

Comment:
if there are missing values in the data, will the function "sum" handle them correctly? nansum if you set the missings to nan?

Reply:
This is a good point to raise. I had thought to use `dropna()` but I didn't--my tests just don't have any values which contain nan's. My thought it using `dropna()` from `pandas` is that we can get weighted means from whatever values are present. So, for example, models would get an Ecosystem mean score even if they don't have Burned Area.